SWIPs/swip-message_to_honey.md

```diff
@@ -26,42 +25,75 @@ This SWIP is part of a series of SWIPs (but can be implemented on it's own). To
 
 ## Specification
 <!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for the current Swarm platform and future client implementations.-->
 * The Swarm source code references the address of a smart-contract which acts as a honey pricing oracle for messages.
 * The price oracle implements the `MsgToHoney` interface (to-be specified) 
-* The `MsgToHoney` contract will be initially owned by an M/N multi-sig of Swarm developers and stakeholders.
-* Upon receiving/sending of a message, a node checks its local database if its cached price is valid. If not, the node will query the price oracle to get the current price. The node will apply the current price to the balance of its peer (see diagram below).
+* The `MsgToHoney` contract will be initially owned by a governance structure of Swarm developers and stakeholders. The governance structure shall be simple to set up initially, but facilitate the possibility for upgrading to a more advanced governance structure. 
+* Nodes pro-actively keep track of updating their local cache of honey-prices.
+* Upon receiving/sending of a message, a node checks its local database if the cached price is valid. If not, the node will query the price oracle to get the current price. Subsequently, the node will apply the current price to the balance of its peer (see diagram below).
 * Nodes expect other nodes to apply the same price as they do themselves. Due to the asynchronicity of the network, this will not necessarily be true around the period that prices are updated, which causes accounting dust (pollen) to be collected. This SWIP does not facilitate a solution for this, as the expected amount of pollen is not high enough to cause disruptions in the network. 
```

Comment:
> This SWIP does not facilitate a solution for this, as the expected amount of pollen is not high enough to cause disruptions in the network.

there seems to be no basis for the statement. how do we justify this claim?

Reply:
I thought we had completely removed the "pollen" stuff, at least I had commented to do so in a previous review. Pollen is poorly defined and not useful in this case. In the real world, pollen is not just dust and actually have a useful purpose.

Reply:
Changed to:
This SWIP does not facilitate a solution for this, as it is not expected that accounting imbalances will be high enough to cause disruptions in the network, but additional analysis on the network has to confirm this statement. 

And `pollen` removed. I agree with that!